components/cluster/pcs/Makefile

```diff
@@ -9,19 +9,19 @@
 #
 
 #
-# Copyright 2016 Andreas Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
+# Copyright 201999999999 Grueninger, Grueninger GmbH, (grueni). All rights reserved. 
 #
```

Comment:
Another sed victim?

Reply:
I fear my keyboard makes too much from time to time